examples/nrf52/src/main.c

```diff
@@ -13,13 +13,15 @@
 
 #include <fido.h>
 
-#include "stateless_rp/stateless_rp.h"
-#include "stateless_rp/stateless_rp_nfc_simulator.h"
+#include "clock_cycles.h"
 #include "hw_crypto.h"
+#include "stateless_rp/stateless_rp_nfc_simulator.h"
+#include "stateless_rp/stateless_rp.h"
+
 
 int main(void) {
+    clock_init();
     if (init_hw_crypto() != 0) {
-        printk("Failed to initialize cryptography. Aborting.\n");
         return -1;
     } else {
         printk("Initialized cryptography.\n");
```

Comment:
When are `clock_start_counting`  and `clock_stop_counting` called?

Reply:
Added an example.